tests/integration/courseSlots.test.js

```diff
@@ -399,22 +421,109 @@ describe('COURSE SLOTS ROUTES - PUT /courseslots/{_id}', () => {
     });
   });
 
-  describe('Other roles', () => {
-    it('should a 200 as user is course trainer', async () => {
-      authToken = await getTokenByCredentials(trainer.local);
+  describe('HOLDING_ADMIN', () => {
+    it('should update intra course slot', async () => {
+      authToken = await getTokenByCredentials(holdingAdminFromOtherCompany.local);
       const payload = { startDate: '2020-03-04T09:00:00.000Z', endDate: '2020-03-04T11:00:00.000Z' };
       const response = await app.inject({
         method: 'PUT',
-        url: `/courseslots/${courseSlotsList[2]._id}`,
+        url: `/courseslots/${courseSlotsList[9]._id}`,
         headers: { Cookie: `alenvi_token=${authToken}` },
         payload,
       });
 
       expect(response.statusCode).toBe(200);
     });
 
-    it('should return 200 as user is client admin from course company', async () => {
-      authToken = await getToken('client_admin');
+    it('should update intra_holding course slot', async () => {
+      authToken = await getTokenByCredentials(holdingAdminFromAuthCompany.local);
+      const payload = {
+        startDate: '2020-03-04T09:00:00',
+        endDate: '2020-03-04T11:00:00',
+        address: {
+          street: '39 rue de Ponthieu',
+          zipCode: '75008',
+          city: 'Paris',
+          fullAddress: '37 rue de Ponthieu 75008 Paris',
+          location: { type: 'Point', coordinates: [2.0987, 1.2345] },
+        },
+      };
+      const response = await app.inject({
+        method: 'PUT',
+        url: `/courseslots/${courseSlotsList[10]._id}`,
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload,
+      });
+
+      expect(response.statusCode).toBe(200);
+    });
+
+    it('should update intra_holding course slot without companies', async () => {
+      authToken = await getTokenByCredentials(holdingAdminFromAuthCompany.local);
+      const payload = {
+        startDate: '2020-03-04T09:00:00',
+        endDate: '2020-03-04T11:00:00',
+        address: {
+          street: '39 rue de Ponthieu',
+          zipCode: '75008',
+          city: 'Paris',
+          fullAddress: '37 rue de Ponthieu 75008 Paris',
+          location: { type: 'Point', coordinates: [2.0987, 1.2345] },
+        },
+      };
+      const response = await app.inject({
+        method: 'PUT',
+        url: `/courseslots/${courseSlotsList[11]._id}`,
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload,
+      });
+
+      expect(response.statusCode).toBe(200);
+    });
+
+    it('should return 403 if try to update intra course slot and company is not in holding', async () => {
```

Comment:
idem ici, l'intitulé est pas tres clair, tu pourrais preciser : "and user company is not in holding"

Reply:
done